src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkAuctionResults.tsx

```diff
@@ -1,9 +1,8 @@
-import { Box, Column, Flex, Join, Message, Spacer, Text } from "@artsy/palette"
+import { Column, Flex, Join, Spacer, Text } from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
 import { MetaTags } from "Components/MetaTags"
 import { ArtistAuctionResultItemFragmentContainer } from "Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem"
 import { extractNodes } from "Utils/extractNodes"
-import { __internal__useMatchMedia } from "Utils/Hooks/useMatchMedia"
```

Comment:
I really wish we could use this, but unfortunately doesn't work with SSR 😢

Reply:
It was copied from another component and not removed cause eslint stays silent 👀

Reply:
That's why we flagged it with `__internal__` 🙂 Probably need something more tho